.github/workflows/playwright.yaml

```diff
@@ -0,0 +1,18 @@
+name: Playwright Setup
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install dependencies for all browsers
+        run: npx playwright install-deps
+      - name: Install dependencies for Chromium only
+        run: npx playwright install-deps chromium
+      - name: Run tests
+        run: npm run test -- --testPathPattern=tests
```

Comment:
I tend to have better luck using yarn over npm. very low priority, but you could switch this to `yarn run test -- --testPathPattern=tests`